SD_Card_Read/SD_Card_Read.ino

```diff
@@ -0,0 +1,56 @@
+#include <SD.h>
+
+const int chipSelectPin = 28;
```

Comment:
Just as a matter of preference, I like to make constants all-caps, like `const float FEET_PER_METER` in the main code.